front/src/modules/timesStops/types.ts

```diff
@@ -0,0 +1,5 @@
+import type { SuggestedOP } from 'modules/trainschedule/components/ManageTrainSchedule/types';
+
+export type PathWaypointColumn = SuggestedOP & {
```

Comment:
I think you will need to deactivate the eslint warning here, because you have only 1 export

Reply:
eslint doesn't do that for types